quickstartKotlin/src/main/java/com/twilio/video/quickstart/kotlin/VideoActivity.kt

```diff
@@ -0,0 +1,840 @@
+                    "[RemoteVideoTrackPublication: sid=${remoteVideoTrackPublication.trackSid}, " +
+                    "enabled=${remoteVideoTrackPublication.isTrackEnabled}, " +
+                    "subscribed=${remoteVideoTrackPublication.isTrackSubscribed}, " +
+                    "name=${remoteVideoTrackPublication.trackName}]")
+            videoStatusTextView.text = "onVideoTrackPublished"
+        }
+
+        override fun onVideoTrackUnpublished(remoteParticipant: RemoteParticipant,
+                                             remoteVideoTrackPublication: RemoteVideoTrackPublication) {
+            Log.i(TAG, "onVideoTrackUnpublished: " +
+                    "[RemoteParticipant: identity=${remoteParticipant.identity}], " +
+                    "[RemoteVideoTrackPublication: sid=${remoteVideoTrackPublication.trackSid}, " +
+                    "enabled=${remoteVideoTrackPublication.isTrackEnabled}, " +
+                    "subscribed=${remoteVideoTrackPublication.isTrackSubscribed}, " +
+                    "name=${remoteVideoTrackPublication.trackName}]")
+            videoStatusTextView.text = "onVideoTrackUnpublished"
+        }
+
+        override fun onAudioTrackSubscribed(remoteParticipant: RemoteParticipant,
+                                            remoteAudioTrackPublication: RemoteAudioTrackPublication,
+                                            remoteAudioTrack: RemoteAudioTrack) {
+            Log.i(TAG, "onAudioTrackSubscribed: " +
+                    "[RemoteParticipant: identity=${remoteParticipant.identity}], " +
+                    "[RemoteAudioTrack: enabled=${remoteAudioTrack.isEnabled}, " +
+                    "playbackEnabled=${remoteAudioTrack.isPlaybackEnabled}, " +
+                    "name=${remoteAudioTrack.name}]")
+            videoStatusTextView.text = "onAudioTrackSubscribed"
+        }
+
+        override fun onAudioTrackUnsubscribed(remoteParticipant: RemoteParticipant,
+                                              remoteAudioTrackPublication: RemoteAudioTrackPublication,
+                                              remoteAudioTrack: RemoteAudioTrack) {
+            Log.i(TAG, "onAudioTrackUnsubscribed: " +
+                    "[RemoteParticipant: identity=${remoteParticipant.identity}], " +
+                    "[RemoteAudioTrack: enabled=${remoteAudioTrack.isEnabled}, " +
+                    "playbackEnabled=${remoteAudioTrack.isPlaybackEnabled}, " +
+                    "name=${remoteAudioTrack.name}]")
+            videoStatusTextView.text = "onAudioTrackUnsubscribed"
+        }
+
+        override fun onDataTrackSubscribed(remoteParticipant: RemoteParticipant,
+                                           remoteDataTrackPublication: RemoteDataTrackPublication,
+                                           remoteDataTrack: RemoteDataTrack) {
+            Log.i(TAG, String.format("onDataTrackSubscribed: " +
```

Comment:
I believe this one is an overhead - string templates, `String#format` and concatenation